ladybug/stat.py

```diff
@@ -64,6 +64,8 @@ def is_data_loaded(self):
     def import_data(self):
         """Import data from a stat file.
         """
+        # set default state to ironpython for very old ironpython (2.7.0)
+        iron_python = True
```

Comment:
Hey @chriswmackey, did you have a case that fails? This is what the other part of the code is supposed to handle:

https://github.com/ladybug-tools/ladybug/blob/9a6cb08dd122e1c3a7782904afb45864596bbf97/ladybug/stat.py#L72-L77

Reply:
Here is the answer to my question:
https://discourse.ladybug.tools/t/error-tauclearsky-component/3495/9